src/widgets/refinement-list/refinement-list.js

```diff
@@ -253,7 +254,7 @@ export default function refinementList(
     ? prefixKeys('show-more-', showMoreConfig.templates)
     : {};
   const searchForValuesTemplates = searchForFacetValues
-    ? searchForFacetValues.templates
+    ? searchForFacetValues.templates || sffvDefaultTemplates
     : {};
```

Comment:
This condition seems a bit complex to me. I'm wondering if something like this is more straightforward: 

```js
const searchForValuesTemplates = searchForFacetValues
  ? searchForFacetValues.templates
    ? searchForFacetValues.templates
    : sffvDefaultTemplates
  : {};
```

It's still a double ternary, but this might be easier to read.